hil/cli/project.py

```diff
@@ -80,6 +82,48 @@ def project_connect_node(project, node):
 @project_node.command(name='remove')
 @click.argument('project')
 @click.argument('node')
-def project_detach_node(project, node):
+@click.option('--force', is_flag=True,
+              help="Disables obm and removes all networks before removing "
+                   "node from project. Requires admin privileges.")
+def project_detach_node(project, node, force):
     """Remove <node> from <project>"""
+    if force:
+        client.node.disable_obm(node)
+        print("Disabled OBM")
+        node_info = client.node.show(node)
+        statuses = []
+        if 'nics' in node_info:
```

Comment:
I'm trying to follow the logic here.  So if `'nics' in node_info` (line 95) returns false, then it skips to line 112 and sets `done = False`.  Then next it would go into the logic that checks if port revert finished successfully.  However, `statuses` will always be empty following this scenario, causing that section after line 119 to be skipped.
Maybe instead then we could just skip to line 129 if line 95 is false?